WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SiteCreationCreatingFragment.java

```diff
@@ -228,11 +230,23 @@ private PreviewWebViewClient loadWebview() {
         @Override
         public void onPageFinished(WebView view, String url) {
             super.onPageFinished(view, url);
-
+            hideGetStartedBar(view);
+            
             mIsPageFinished = true;
         }
     }
 
+    // Hacky solution to https://github.com/wordpress-mobile/WordPress-Android/issues/8233
+    // Ideally we would hide "get started" bar on server side
+    @SuppressLint("SetJavaScriptEnabled")
+    private static void hideGetStartedBar(@NonNull WebView webView) {
+        webView.getSettings().setJavaScriptEnabled(true);
+        String javascript = "document.querySelector('html').style.cssText += '; margin-top: 0 !important;';\n"
+                            + "document.getElementById('wpadminbar').style.display = 'none';\n";
+
+        webView.evaluateJavascript(javascript, null);
```

Comment:
Do you think we should disable JS after the changes are made to the doc? 
(Just for a security reason).

Reply:
Good call!